src/js/providers/html5.js

```diff
@@ -443,8 +446,11 @@ define([
                 return;
             }
 
-            _setLevels(item.sources);
-            this.sendMediaType(item.sources);
+            // check if _setLevels has already been called
```

Comment:
The on('levels') count increases in videoEventHandler in model (this.mediaController.trigger(type, evt);), so I am not sure how we can filter this.
Would it cause problems if we don't send levels event on init? It will be called in load anyways when the user tries to play the video.